src/simple_transformation.py

```diff
@@ -0,0 +1,20 @@
+from talon import Context, Module
```

Comment:
Let's remove this file.  It's only used in `matching_range_transformation`, which isn't used anywhere